rocAL/include/pipeline/commons.h

```diff
@@ -211,11 +220,12 @@ class BatchRNG {
 /*! \brief MissingComponentsBehaviour for Webdataset
  *
  */
-enum MissingComponentsBehaviour {
+enum class MissingComponentsBehaviour {
     MISSING_COMPONENT_ERROR = 0,
     MISSING_COMPONENT_SKIP,
     MISSING_COMPONENT_EMPTY
 };
+REGISTER_ENUM(MissingComponentsBehaviour)
```

Comment:
Non-blocking: Associated enum is defined as . Consider explicitly specifying the underlying type (e.g., ) to keep ABI stable and avoid surprises across compilers.